spec/labor/bufferizer_spec.rb

```diff
@@ -19,10 +19,10 @@
   end
 
   it "sends to buffer sattelite twitter" do
-    tweet = "test tweet #DEVCommunity"
+    tweet = "test tweet #{SiteConfig.twitter_hashtag}"
```

Comment:
I suggest setting `SiteConfig.twitter_hashtag` explicitly in tests before performing checks that include it.  It would be nice not to rely on the default value, especially regarding that we plan to remove the default values in the future, so may accidentally keep tests that check for empty strings then.

Reply:
@lightalloy Thanks for the tip!